recipes/flask-jwt-extended/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "Flask-JWT-Extended" %}
+{% set version = "3.3.1" %}
+{% set sha256 = "fe25451a67641ac3007ebbde575e35dfbda7f95b95434b65ff10e3d37f2986b7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - flask >=0.12.2
+    - pyjwt >=1.5.2
+    - python
+
+test:
+  imports:
+    - flask-jwt-extended
+
+about:
+  home: https://github.com/vimalloc/flask-jwt-extended
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A Flask JWT extension'
+
+  description: |
+    A Flask JWT extension that supports refresh tokens, blacklisting/revoking tokens,
+    and token freshness (for accessing critical views)
+
+extra:
+  recipe-maintainers:
+    - lvoliveira
+    - nicoddemus
```

Comment:
@nicoddemus and @igortg -- are you ok being added as co-maintainers of the flask-jwt-extended recipe on conda-forge?

Reply:
👍 yes thanks

Reply:
Yep